common/pulp/common/constants.py

```diff
@@ -13,3 +13,9 @@
 TASK_FINAL_STATES = (TASK_STATES.SKIPPED, TASK_STATES.COMPLETED, TASK_STATES.FAILED,
                      TASK_STATES.CANCELED)
 TASK_INCOMPLETE_STATES = (TASK_STATES.WAITING, TASK_STATES.RUNNING)
+
+# The interval in seconds for which the Celery Process monitor thread sleeps between
+# checking for missing Celery processes.
+CELERY_CHECK_INTERVAL = 60
```

Comment:
I think leaving them hard-coded would be OK for now. If we make them settings then it's possible users could get undesirable behaviors if they aren't set correctly. I'm not ready to write all those docs so I propose leaving it hard-coded for now.